devices/src/legacy/fw_cfg.rs

```diff
@@ -42,6 +54,13 @@ struct boot_params {
 // SAFETY: boot_params is only data, reading it from data is a safe initialization.
 unsafe impl ByteValued for boot_params {}
 
+#[cfg(target_arch = "x86_64")]
+const STAGE0_START_ADDRESS: GuestAddress = GuestAddress(0xffe0_0000);
+#[cfg(target_arch = "x86_64")]
+const STAGE0_SIZE: usize = 0x20_0000;
```

Comment:
Can you please provide some pointers to how these values are defined? Are these specific to Oak firmware? I may miss some background knowledge in general.